helm-chart/values.yaml

```diff
@@ -57,12 +57,22 @@ nginxLBMethod: "least_time"
 # Setting to "0" disables checking of client request body size.
 clientMaxBodySize: "1m"
 
+# Globally disable automatic sidecar injection upon resource creation.
+# Use either "enabledNamespaces" or a namespace label to enable automatic injection.
+disableAutoInjection: false
+
+# Enable automatic sidecar injection for specific namespaces.
+# Must be used with "disableAutoInjection".
+enabledNamespaces: []
+
 # The address of a Prometheus server deployed in your Kubernetes cluster.
 # Address should be in the format <service-name>.<namespace>:<service-port>.
 prometheusAddress: ""
 
+# This block has been deprecated and will be removed in a future release of NGINX Service Mesh. Please use top level values disable and enabledNamespaces.
```

Comment:
> Please use top level values disable and enabledNamespaces.

This sentence doesn't make sense. Should it be `'disableAutoInjection' and 'enabledNamespaces'`?